x-pack/solutions/observability/plugins/apm/common/agent_configuration/setting_definitions/edot_sdk_settings.ts

```diff
@@ -66,7 +67,7 @@ export const edotSDKSettings: RawSettingDefinition[] = [
       { text: 'fatal', value: 'fatal' },
       { text: 'off', value: 'off' },
     ],
-    includeAgents: ['opentelemetry/java/elastic'],
+    includeAgents: [...EDOT_AGENT_NAMES],
```

Comment:
Not sure if it's fine to advertise this for all agents, Python does not support this yet

Reply:
I'll change this to be just the java and nodejs EDOT agent names for now. (I had mistakenly assumed that all the EDOT SDKs would support this setting.)